django-stubs/db/migrations/state.pyi

```diff
@@ -7,8 +7,11 @@ from django.apps.registry import Apps
 from django.db.models.base import Model
 from django.db.models.fields import Field
 from django.db.models.manager import Manager
+from django.utils.functional import cached_property
 
-class AppConfigStub(AppConfig): ...
+class AppConfigStub(AppConfig):
+    def __init__(self, label: str) -> None: ...
+    def import_models(self) -> None: ...
```

Comment:
This method exists in the base class with the same signature?
```suggestion
```

Reply:
Good catch thanks :)

This kind of testing would actually be very nice, to identify useless overrides!